src/main/java/uk/gov/pay/api/model/AuthorisationRequest.java

```diff
@@ -0,0 +1,49 @@
+package uk.gov.pay.api.model;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import javax.validation.constraints.NotBlank;
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Size;
+
+public class AuthorisationRequest {
+    @JsonProperty("one_time_token")
+    @NotNull
```

Comment:
Any reason `@NotNull` is used for all expect `cardholderName` which has `@NotBlank` annotation ?

Reply:
I've changed `oneTimeToken` to NotBlank also. The others has a minimum field size thus they only need null check.